src/main/java/org/dbos/apiary/mysql/MysqlContext.java

```diff
@@ -63,8 +73,43 @@ private void prepareStatement(PreparedStatement ps, Object[] input) throws SQLEx
         }
     }
 
+    // Note: this executeUpdate should only be used by non-XDST transactions.
```

Comment:
Why isn't this done the same way as in Mongo, where there's just one function with a flag?

Reply:
The issue is that XDST cannot support arbitrary SQL `UPDATE` queries that have some predicates. This function is mainly designed for non-XDST benchmarks.

Reply:
I guess, but do we use it anywhere?

Reply:
Yes, I use it here [MysqlReplacePerson.java](https://github.com/DBOS-project/apiary/pull/108/files#diff-e55cf2c72c5e04a951a7692827835965d796e1033d793fb09a8819df33dde8f3). Because I want to benchmark non-XDST update, which is supposed to be faster than upsert (insert on conflict in MySQL).